src/data/ha-state-history-data.html

```diff
@@ -104,7 +103,9 @@
       this._setIsLoading(true);
 
       data.then(function (stateHistory) {
-        this._setData(stateHistory);
+        // Dom restoration is async and if graph is drown before dom is
```

Comment:
Already removed from here.

I do thing `more-info` got the fix wrong.

history panel had no dom restoration because ` is-loading` was misspelled as ` isLoading`
Once I fixed that the timeline started to be 400px after changing data.

We could remove the loader when changing data (only keep it at the initial load). I do think the loader is nice, especially now that we are fetching larger amounts of data.